src/scf/self_consistent_field.jl

```diff
@@ -180,11 +180,15 @@ Overview of parameters:
         if compute_consistent_energies
             (; energies) = energy(basis, ψ, occupation; ρ=ρout, eigenvalues, εF)
         end
+
+        # Update info with history
         history_Etot = vcat(info.history_Etot, energies.total)
         history_Δρ = vcat(info.history_Δρ, norm(Δρ) * sqrt(basis.dvol))
+        history_εF = vcat(info.history_εF, εF)
         n_matvec = info.n_matvec + nextstate.n_matvec
-        info_next = merge(info_next, (; energies, history_Etot, history_Δρ, n_matvec))
-
+        info_next = merge(info_next, (; energies, history_Etot, history_Δρ, history_εF,
+            n_matvec))
```

Comment:
Here the indention is weird, I'd do
```suggestion
        info_next = merge(info_next, (; energies, history_Etot, history_Δρ, history_εF,
                                        n_matvec))
```